vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,10 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted] | ::Vmdb::ConfigurationEncoder::PASSWORD_FIELDS).sort
```

Comment:
So, are we returning the config itself or a copy of it?  Why would the REST API return it at all?